### ISSUE TITLE
Release notes

### ISSUE THREAD

**Issue by frenesim** [Shopify/maintenance_tasks#917]
@elfassy 

The release for [v2.3.0](https://github.com/Shopify/maintenance_tasks/releases/tag/v2.3.0) it adds metadata column.
On the notes it's required to run new migrations and use `bin/rails maintenance_tasks:install:migrations`

In the code bases that had archived migrations it will generate all the migrations and one may became in doubt which ones one needs.

Maybe it's worth to add a note saying that only the `add_metadata_to_runs.maintenance_tasks` migration is needed 🤔 

I'd be happy to add the note, but I couldn't find an easy away to do that.

Thanks for the gem ❤️

**Comment by etiennebarrie** [comment]
Yeah we have a section for this in the [README](https://github.com/Shopify/maintenance_tasks#upgrading:~:text=What%20if%20I%E2%80%99ve%20deleted%20my%20previous%20Maintenance%20Task%20migrations%3F), but I'll update the release notes to include a word about the migration.
